lib/forest_liana/bootstraper.rb

```diff
@@ -522,6 +539,61 @@ def stripe_integration_deprecated?
       is_deprecated
     end
 
+    def setup_mixpanel_integration(collection_name_and_field)
+      collection_name = collection_name_and_field.split('.')[0]
+      model_name = ForestLiana.name_for(collection_name.constantize)
+      collection_display_name = model_name.capitalize
+
+      field_attributes = { 'is-filterable': false , 'is-virtual': true, 'is-sortable': false }
+
+      fields = [
+        { field: :id, type: 'String' },
+        { field: :event, type: 'String' },
+        { field: :date, type: 'Date' },
+        { field: :city, type: 'String' },
+        { field: :region, type: 'String' },
+        { field: :country, type: 'String' },
+        { field: :timezone, type: 'String' },
+        { field: :os, type: 'String' },
+        { field: :osVersion, type: 'String' },
+        { field: :browser, type: 'String' },
+        { field: :browserVersion, type: 'String' },
+      ]
+
+      fields = fields.map { |field| field.merge(field_attributes) }
+
+      custom_properties = ForestLiana.integrations[:mixpanel][:custom_properties]
+      custom_properties = custom_properties.map { |property|
+        { field: property.to_sym, type: 'String', 'is-filterable': false }
```

Comment:
Does not seem that you applied `field_attributes` to those custom property fields.